config/winston.js

```diff
@@ -1,24 +1,27 @@
-const config = require('./config');
+const { configuredFormatter } = require('winston-json-formatter');
 
-const { createLogger, transports, format } = require('winston');
-
-const { printf, timestamp, combine, colorize } = format;
+const { createLogger, transports } = require('winston');
 
 const logger = createLogger({
-    level: config.logLevel,
     transports: [
         new transports.Console(),
     ],
 });
 
-const developmentFormat = printf(info => `${info.timestamp} ${info.level}: ${info.message}`);
+const options = {
+    service: 'test-service',
+    name: 'Winston-JSON-Formatter',
+    version: '1.0.0',
+    typeFormat: 'json',
```

Comment:
Don't explicitly set `typeFormat`. Allow it to be set dynamically by `NODE_ENV`

Reply:
@rmharrison when I created the first iteration of the amida winston logger in the messaging service we added an env variable of LOG_LEVEL. It is unclear to me if I should be passing this var into the options object when called configuredFormatter or if this should be handled somewhere else.

Reply:
^^^ sorry if this is unclear. I am adding the type format env variable but had this question about the existing env variable I had added to the services

Reply:
Resolved in pair. *Indeed* `NODE_ENV` because it sets `typeFormat=console` (dev) vs `typeFormat=json` (all others)